src/containers/booksform.js

```diff
@@ -1,14 +1,41 @@
 /* eslint-disable react/no-unused-state */
 /* eslint-disable react/prefer-stateless-function */
 import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
 import { v4 as uuidv4 } from 'uuid';
+import { addBook } from '../actions/index';
 
 class BooksForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      title: '',
+      category: '',
+    };
+    this.myRef = React.createRef();
+    this.handleChange = this.handleChange.bind(this);
+    this.handleDrop = this.handleDrop.bind(this);
+  }
+
+  handleDrop() {
+    this.setState({ title: this.inputNode.value, category: this.myRef.current.value });
+  }
+
+  handleChange(e) {
+    e.preventDefault();
+    // eslint-disable-next-line react/destructuring-assignment
```

Comment:
- Don't disable linter warnings. Fix every warning/errors caused by the linter. Most of them can be fixed.